components/blog/PageHeader.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+interface PageHeaderProps {
+	heading: string,
+	text?: string,
+}
+
+const PageHeader = ({ heading, text }: PageHeaderProps) => {
+	return (
+		<div className="my-16 w-full text-center">
+			{text && <span className="text-violet-400 font-bold">{text}</span>}
```

Comment:
In the next PR, let's replace all such customer colors definitions with our own color pallet.